dev/com.ibm.ws.crypto.passwordutil/src/com/ibm/ws/crypto/util/AESKeyManager.java

```diff
@@ -78,6 +87,14 @@ private KeyHolder get(char[] keyChars) {
         }
     }
 
+    // Used exclusively for debugging securityUtility
+    private static void securityUtilDebug(String message, Throwable throwable) {
+        if (DEBUG) {
```

Comment:
Is the DEBUG check here necessary, if we are checking if DEBUG is enabled before calling this method? Unless you don't want each caller to check If debug is enabled, it would be cleaner to add it here and remove the above checks?

Reply:
Makes sense